.gitignore

```diff
@@ -51,6 +51,9 @@ coverage.xml
 .hypothesis/
 .pytest_cache/
 
+# Threshold values
+files\image_processing\threshold_values.txt
```

Comment:
@Holandsoest Do the backslashes in the gitignore also need to be changed?

Reply:
Great question!

This will be addressed in the [Issue: 19-path-fix](https://github.com/Holandsoest/art-to-music/issues/19)

Reply:
I searched and according to [stackoverflow](https://stackoverflow.com/questions/51263778/forward-slash-in-gitignore-files) Git works with forward-slashes only `/` so i don't even know why it worked for you...  
As mentioned by @Holandsoest it will be adressed in Issue 19